lib/util/versioning.js

```diff
@@ -1,6 +1,19 @@
+const { isMasterKey } = require('arsenal').versioning;
 const VID_SEP = require('arsenal').versioning.VersioningConstants
           .VersionId.Separator;
 
+/**
+ * @param {string} key full object key (_id in metadata)
+ * @returns {srting | null} version id
+ */
+function extractVersionId(key) {
+    if (key.includes(VID_SEP)) {
+        return key.split(VID_SEP)[1];
+    }
+    return null;
+}
+
 module.exports = {
-    isMasterKey: key => !key.includes(VID_SEP),
+    isMasterKey,
```

Comment:
is this export coming from Arsenal done on purpose?

Reply:
That function is actually no longer used in backbeat, i ran e2e tests and everything seems to still work, i'm removing it